main.md

```diff
@@ -236,21 +235,17 @@ GET /authorize?response_type=code&client_id=s6BhdRkqt3
     &redirect_uri=https%3A%2F%2Fclient.example.org%2Fcb 
     &code_challenge_method=S256
     &code_challenge=K2-ltc83acc4h0c9w6ESC_rEMTJ3bww-uCHaoeK1t8U
-    &authorization_details=%5B%7B%22type%22%3A%22https%3A%2F%2F
-    www.someorg.com%2Fpayment_initiation%22%2C%22actions%22%3A%5
-    B%22initiate%22%2C%22status%22%2C%22cancel%22%5D%2C%22locat
-    ions%22%3A%5B%22https%3A%2F%2Fexample.com%2Fpayments%22%5D%
-    2C%22instructedAmount%22%3A%7B%22currency%22%3A%22EUR%22%2C
-    %22amount%22%3A%22123.50%22%7D%2C%22debtorAccount%22%3A%7B%
-    22iban%22%3A%22DE40100100103307118608%22%7D%2C%22creditorNa
-    me%22%3A%22Merchant123%22%2C%22creditorAccount%22%3A%7B%22i
-    ban%22%3A%22DE02100100109307118603%22%7D%2C%22remittanceInf
-    ormationUnstructured%22%3A%22Ref%20Number%20Merchant%22%7D%
-    5D%0A%20%20%20 HTTP/1.1
+    &authorization_details=%5B%7B%22type%22:%22account_information
+    %22,%22actions%22:%5B%22list_accounts%22,%22read_balances%22,%
+    22read_transactions%22%5D,%22locations%22:%5B%22https://exampl
+    e.com/accounts%22%5D%7D%5D HTTP/1.1
```

Comment:
Looks like the full authz deets JSON is included now. So the `(JSON string trimmed for brevity)` part could and should be removed from the text just before this example.

Reply:
done